Userland/Utilities/pgrep.cpp

```diff
@@ -50,9 +52,18 @@ ErrorOr<int> serenity_main(Main::Arguments args)
 
     quick_sort(matches);
 
+    auto displayed_at_least_one = false;
     for (auto& match : matches) {
-        outln("{}", match);
+        if (displayed_at_least_one)
+            out("{}{}"sv, pid_delimiter, match);
+        else
+            out("{}"sv, match);
+
+        displayed_at_least_one = true;
     }
 
+    if (displayed_at_least_one)
```

Comment:
This feels a bit awkward... why do we do this?

Reply:
This ensures that the output ends with a newline, even when we don't use the default newline delimiter. `pidof`  has the same logic.

Reply:
Sure, that makes sense :)